### PR TITLE
projects: ad9434: add IIO support

### PR DESCRIPTION
**Pull request by amiclaus**
Allow the user to read/write via UART using IIO oscilloscope.

Signed-off-by: Antoniu Miclaus <antoniu.miclaus@analog.com>

### CODE REVIEW

**Comment by amiclaus** [comment]
V2:
- remove redundant `ifndef`

File: projects/ad9434-fmc-500ebz/src/ad9434_fmc_500ebz.c [hunk 1]
```diff
@@ -147,6 +153,20 @@ int main(void)
 		return FAILURE;
 	}
 
+#ifdef IIO_SUPPORT
+	printf("The board accepts libiio clients connections through the serial backend.\n");
```
Comment (mchindri) [inline projects/ad9434-fmc-500ebz/src/ad9434_fmc_500ebz.c:157]: Use print_info or print_dbg

**Review by amiclaus — COMMENTED** [review]
V2:
- use `pr_info` instead of `printf`

**Review by adrimbarean — APPROVED** [review]
Looks good to me.

**Pull request merged**